### PR TITLE
fix: drop tape

### PR DESCRIPTION
**Pull request by lili2311**
- [ ] Ready for review
- [ ] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?

Drop `tape` and use `tap` instead

### CODE REVIEW

File: test/deep-deps.test.ts [hunk 1]
```diff
@@ -17,6 +17,6 @@ test('finds all sub-dependencies', async (t) => {
   } catch (e) {
     t.fail(e.message);
     console.log(e.stack);
-    t.bailout();
+    t.end();
```
Comment (gitphill) [inline test/deep-deps.test.ts]: do you need end (async function) ? could also remove console.log or put whole error object in fail if it's useful

Comment (lili2311) [reply test/deep-deps.test.ts]: you are right no need to end, will update

File: test/no-deps.test.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+const osDir = path.resolve(__dirname, 'fixtures', 'no-deps');
+
+test('works when there are no dependencies', async (t) => {
+  try {
+    const modules = await snyk.modules(osDir);
+    t.ok(true, 'modules did not bail');
+    t.deepEqual(modules.dependencies, {});
+  } catch (e) {
+    console.log(e.stack);
+    t.fail(e.message);
```
Comment (gitphill) [inline test/no-deps.test.ts]: fail with e object and remove log?

File: test/policy-populates-prompt.test.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+    t.notEqual(rule.reason, undefined, 'reason should not be undefined');
+    t.equal(
+      prompt.default,
+      rule.reason,
+      'reasons should match in default value',
+    );
+  } catch (e) {
+    console.log(e.stack);
+    t.fail('could not load the policy file');
```
Comment (gitphill) [inline test/policy-populates-prompt.test.ts]: fail with e object and remove log?

File: test/scoped-module.test.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-var test = require('tape');
+var test = require('tap').test;
```
Comment (lili2311) [inline test/scoped-module.test.js]: did not convert these as the diff was getting huge, will come back to them

File: test/protect-patch.test.js [hunk 5]
```diff
@@ -2,7 +2,7 @@
 var debug = require('debug')('snyk');
 var protect = require('../src/lib/protect');
 var path = require('path');
-var test = require('tape');
+var test = require('tap').test;
```
Comment (aviadhahami) [inline test/protect-patch.test.js]: `var` ? :owl:

**Pull request merged**